src/main/resources/embedding-providers-config.yaml

```diff
@@ -104,6 +104,46 @@ stargate:
                   help: "Vector dimension to use in the database and when calling Azure OpenAI."
             - name: text-embedding-ada-002
               vector-dimension: 1536
+        bedrock:
+          display-name: Amazon Bedrock
+          enabled: true
+          url: " "
```

Comment:
Maybe add YAML comment explaining that URL is handled by client library, so no value to add here?

Reply:
Create an issue to fix it later: https://github.com/stargate/data-api/issues/1231